cmd/node/factory/structs.go

```diff
@@ -1240,11 +1241,22 @@ func newShardBlockProcessorAndTracker(
 		return nil, nil, err
 	}
 
+	// TODO: construct this correctly on the PR
+	specialAddressHolder, err := address.NewSpecialAddressHolder(
```

Comment:
it will be linked with nodesCoordinator and the PR Sebi is doing with the addresses in the config. It is used to send the special transaction with the fees to the set addresses.

Reply:
ok